app/views/layouts/_application_content.html.erb

```diff
@@ -15,17 +15,18 @@
         <%= mount_breadcrumbs %>
       <% end %>
     </div>
-      <div class="row">
-        <div class="title_filter <%= searchable? ? " col-md-6 " : "col-md-4 " %>">
-          <%= render "common/searchbar" if searchable? %>
-            <%= yield(:search_bar) %>&nbsp;
-        </div>
-        <div id="title_action" class= <%= searchable? ? "col-md-6" : "col-md-8" %>>
-          <div class="btn-toolbar pull-right">
-            <%=h yield(:title_actions) %>
-          </div>
+    <%= yield(:before_search_bar) %>
```

Comment:
This is the only line added, the other changes are just fixing the incorrect spaces